st2actions/st2actions/notifier.py

```diff
@@ -120,12 +122,17 @@ def _post_generic_trigger(self, liveaction):
                    'status': liveaction.status,
                    'start_timestamp': str(liveaction.start_timestamp),
                    'action_name': liveaction.action,
```

Comment:
It will break https://github.com/StackStorm/st2incubator/blob/master/packs/st2cd/rules/st2_docs_prod.yaml#L8 and many more.

action_name is from before ref and now we have backward compatibilty to deal with.

I can add a new property `action_ref` and slowly deprecate `action_name`.